DesktopModules/AgapeConnect/Stories/ViewStory.ascx.vb

```diff
@@ -138,10 +141,15 @@ Namespace DotNetNuke.Modules.FullStory
             Dim URL = StoryFunctions.GetPhotoURL(story.PhotoId)
             Dim Fid = StaffBrokerFunctions.GetSetting("FacebookId", PortalSettings.PortalId)
             Dim permalink = NavigateURL(TabId, "", GetStoryURLParams(story.StoryId, Request.QueryString(ORIGINAL_MODULEID), Request.QueryString(ORIGINAL_TABID)))
-
+            If ((story.Latitude IsNot Nothing) And (story.Longitude IsNot Nothing)) Then
+                latitude = story.Latitude.Value.ToString(New CultureInfo(""))
+                longitude = story.Longitude.Value.ToString(New CultureInfo(""))
+                location = latitude & "," & longitude
+            End If
```

Comment:
I would move everything in this If below (after ReplaceField(template, "[LANGUAGES]", "")) with a comment line and its own space since there is logic here. Move the Dim maphtml... with it too. 

This section is more for straight field mapping no logic. Will keep the code cleaner.

Reply:
Cleaned up the code. Needs to come before replacements as it contains necessary logic for replacements.